Source/DataSources/KmlDataSource.js

```diff
@@ -2369,6 +2391,9 @@ define([
                             //Insert missing namespaces
                             text = insertNamespaces(text);
 
+                            //Remove Duplicate Namespaces
+                            text = removeDuplicateNamespaces(text);
```

Comment:
This won't get called for kmz files. Look for all the occurrences of `insertNamespaces(...)`. This should be called in the same places.

Reply:
Thank you for finding that. I have updated the PR with a call for KMZ's as well. Please let me know if you find any other issues. Thank you.